build.gradle

```diff
@@ -67,6 +67,14 @@ subprojects {
         }
     }
 
+    // Ensure that Javadoc generation is always tested
+    check.dependsOn(javadoc)
+
+    javadoc {
```

Comment:
@bsideup I think we probably should have had this all along, so that Javadocs are generated based on delomboked sources.

For some reason that I can't understand, the Azure Linux build was failing without this - I've not looked into why yet.